src/blockchain/shelley/utils.js

```diff
@@ -104,8 +106,8 @@ const fragmentToObj = (fragment: any, extraData: {}): TxType => {
         const deleg = cert.get_stake_delegation()
         const poolId = deleg.delegation_type().get_full()
         common.certificate = {
-          type: 'StakeDelegation',
-          // TODO: possibly handle Ratio types
+          type: wasm.CertificateType.StakeDelegation,
+          // TODO: handle DelegationType parsing
```

Comment:
according to this comment: https://github.com/Emurgo/tangata-manu/pull/99/files#r350673514

Here type should also be changed back to `type: 'StakeDelegation'`